src/main/java/banking/Menu.java

```diff
@@ -0,0 +1,168 @@
+package banking;
+
+import banking.database.Operations;
+import banking.processing.Account;
+import banking.processing.Card;
+
+import java.util.Scanner;
+
+import static java.lang.System.exit;
+
+public class Menu {
```

Comment:
Can you rename the methods of this class?

Reply:
fixed